src/components/Boards/Boards.tsx

```diff
@@ -0,0 +1,77 @@
+import { Grid } from "@material-ui/core";
+import { Component } from "react";
+import { connect } from "react-redux";
+import "components/Boards/Boards.css";
+import { Status } from "common/constants";
+import Board from "components/Board/Board";
+import { RootState } from "store/store";
+import { IIssue } from "common/models";
+import CreateIssue from "components/Issue/CreateEditIssue/CreateIssue/CreateIssue";
+
+type Acc = {
```

Comment:
Acc might not be the best naming. Better have longer type name but self explaining, rather than shorter one which is not self explaining

Reply:
done